code/modules/modular_computers/file_system/programs/engineering/_engineering.dm

```diff
@@ -40,6 +41,7 @@
 	extended_desc = "This program provides visual interface for station's alarm system."
 	requires_ntnet = TRUE
 	network_destination = "alarm monitoring network"
+	usage_flags = PROGRAM_CONSOLE | PROGRAM_LAPTOP | PROGRAM_TABLET | PROGRAM_STATIONBOUND | PROGRAM_WRISTBOUND
```

Comment:
Why add restrictions? Also please do not list every type and use agregte type `PROGRAM_ALL_REGULAR` with agregite type `PROGRAM_STATIONBOUND`

Reply:
Already mentioned in the above review.